src/libraries/Microsoft.PowerFx.Core/Binding/BindingConfig.cs

```diff
@@ -19,10 +20,13 @@ internal class BindingConfig
 
         public bool UseThisRecordForRuleScope { get; }
 
-        public BindingConfig(bool allowsSideEffects = false, bool useThisRecordForRuleScope = false)
+        public CultureInfo CultureInfo { get; }
```

Comment:
> CultureInfo CultureInfo { get; } [](http://example.com/codeflow?start=14&length=33)

This is explicitly for error messages? Has no other impact on binding.  (unlike parse modes)

Reply:
Removed.